### DISCUSSION TITLE
How to start sunshine on boot

### DISCUSSION THREAD

**Discussion by WesleyVanNeck** [LizardByte/community#344] [Sunshine]
I have trouble with starting sunshine on wayland without sudo and sudo setcap cap_sys_admin+p $(readlink -f $(which sunshine)) doesn't work on flatpak version any idea how fix it

**Comment by spoofer8** [comment]
I also would like to do that but for windows

**Reply by ReenigneArcher** [reply]
It's automatic with the installer.

**Comment by ReenigneArcher** [comment]
The docs list a required additional install step for our flatpak.

**Reply by WesleyVanNeck** [reply]
Yeah but it doesn't work

**Reply by ReenigneArcher** [reply]
I'm guessing you are using flathub version instead of OUR version

**Reply by WesleyVanNeck** [reply]
No I use the latest release from the GitHub page

[a comment by an automated account is not shown]
